### ISSUE TITLE
"Adding Selected Page" appears unresponsive

### ISSUE THREAD

**Issue by pandafulmanda** [Connexions/webview#961]
There is a significant lag time between clicking "Add Selected Page(s)" and the modal hiding without indication of any progress.  I think that this will confuse users and cause them to click the button multiple times.

Also, it would nice if when the modal closes, the window scrolls down to the new page.

**Comment by amberwebb** [comment]
@pandafulmanda I don't see a lag time. How many pages have you got checked? Is it happening in a certain browser?

**Comment by pandafulmanda** [comment]
Just one.  I've seen this happen in both Chrome and Firefox now.  I guess by "significant" I mean, like...1 or 2 seconds.  That might not be a lot of time, but it was enough time that since I did not see anything change immediately when I clicked "Add Selected Page(s)," I thought something was wrong the first few times I did it, especially since the page seizes and I am not able to scroll for example.

Also, since the activity indicator immediately shows when I click Search and the modal immediately closes when I just "Create a new page" as opposed to adding a search result makes the wait time stand out more.

Maybe the lag is a result of of some setting I have?
